core/arch/arm/plat-bcm/conf.mk

```diff
@@ -0,0 +1,35 @@
+PLATFORM_FLAVOR ?= ns3
+
+ifeq ($(CFG_ARM64_core),y)
+$(call force,CFG_WITH_LPAE,y)
+platform-flavor-armv8 := 1
```

Comment:
I suppose this should rather be set based on `PLATFORM_FLAVOR`

Reply:
yes. It is more apt to be under PLATFORM_FLAVOR.  I will do this change.